ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml

```diff
@@ -96,6 +96,16 @@ tests:
   steps:
     cluster_profile: ibmcloud-qe
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-proxy-private
+- as: installer-rehearse-vsphere-discon-proxy
+  cron: '@yearly'
+  steps:
+    cluster_profile: vsphere-dis
+    workflow: cucushift-installer-rehearse-vsphere-ipi-proxy
+- as: installer-rehearse-vsphere-discon-upi
```

Comment:
I think we only need to add one `installer-rehearse-vsphere-discon` job here for a simple testing, wdyt?

Reply:
yes, I can remove one

Reply:
@sgaoshang suggest to update the job name to `installer-rehearse-vsphere-dis`

Reply:
Updated, thanks.